Backpack-SwiftUI/ImageGallerySlideshow/Classes/BPKImageGallerySlideshow.swift

```diff
@@ -0,0 +1,180 @@
+/*
+ * Backpack - Skyscanner's Design System
+ *
+ * Copyright 2018 Skyscanner Ltd
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import SwiftUI
+
+struct ImageGallerySlideshow<ImageView: View>: ViewModifier {
+    let images: [BPKImageGalleryImage<ImageView>]
+    let closeAccessibilityLabel: String
+    let onCloseTapped: () -> Void
+    @Binding var currentIndex: Int
+    @Binding var isPresented: Bool
+    
+    func body(content: Content) -> some View {
+        content
+            .fullScreenCover(isPresented: $isPresented, content: {
+                ContentView(
+                    images: images,
+                    closeAccessibilityLabel: closeAccessibilityLabel,
+                    onCloseTapped: onCloseTapped,
+                    currentIndex: $currentIndex
+                )
+            })
+    }
+    
+    struct ContentView: View {
+        let images: [BPKImageGalleryImage<ImageView>]
+        let closeAccessibilityLabel: String
+        let onCloseTapped: () -> Void
+        @Binding var currentIndex: Int
+        
+        var body: some View {
+            GeometryReader { proxy in
+                VStack(alignment: .leading, spacing: .xl) {
+                    header
+                    
+                    ZStack(alignment: .bottom) {
+                        InternalCarouselWrapper(
+                            images: images.map { $0.content() },
+                            pageIndicatorVisibility: .hidden,
+                            currentIndex: $currentIndex
+                        )
+                        BPKBadge("\(currentIndex + 1)/\(images.count)")
+                            .badgeStyle(.strong)
+                            .padding(.bottom, 20)
```

Comment:
should this use BPKSpacing values?

Reply:
There's no BPKSpacing with a value of 20, both 16 or 24 are too far from 20 to make sense using :/